config/settings.yml

```diff
@@ -1,3 +1,33 @@
+# the first endpoint_type listed must be for an local "online"
+# endpoint, and the default values listed for it will be used
+# in combination with the defined storage_roots to seed the db
+# with endpoints.
+endpoint_types:
+  local_online_nfs:
+    endpoint_class: online
+    endpoint_defaults:
+      endpoint_node: 'localhost'
+      recovery_cost: 1
+
+# TODO: more illustrative fixture setup: create two storage roots, 'spec/fixtures/storage_root_1' and
+# 'spec/fixtures/storage_root_2', and split the moab fixtures between them.  also, rename 'moab_storage_root'
+# to 'moab_storage_trunk'.
```

Comment:
created a [ticket](https://github.com/sul-dlss/preservation_core_catalog/issues/140), will get rid of this comment